src/vm/arm/asmhelpers.asm

```diff
@@ -1653,7 +1653,7 @@ __ProfilerHelperFunc SETS "$HelperName":CC:"Naked"
         ldr     r3, [r0, #(MachState___R4_R11 + (($regIndex-4)*4))]
 
         cmp     r2, r3
-        bne     %FT0
+        bne     0f
```

Comment:
Modifications to this file were in error, I'll revert them. Must of find/replaced with too many files open.